.github/workflows/ci-cd.yaml

```
@@ -26,11 +26,11 @@ jobs:
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.6.1
       - name: Lint charts
-        run: ct lint --charts=charts/trino --validate-maintainers=false
+        run: ct lint --charts=charts/trino,charts/gateway --validate-maintainers=false
 
   test:
     runs-on: ubuntu-latest
-    name: test ${{ matrix.label }}
+    name: trino-test ${{ matrix.label }}
```

Make it consistent with the `test gateway` job:
```suggestion
    name: test trino ${{ matrix.label }}
```